FogCreek/Glitch-Community
# Fallacious slice: Force enable/disable optimizely flags

## Links
https://fallacious-slice.glitch.me/secret
https://app.clubhouse.io/glitch/story/1593/a-b-testing-with-optimizely

## GIF/Screenshots:
![image](https://user-images.githubusercontent.com/8932174/68686227-4e1cb380-0539-11ea-8553-ae07967a12e5.png)

## Changes:
- Added the ability to force enable/disable an optimizely flag for all user ids
- The secret page lists all features and a dropdown to enable/disable/restore original behavior

## How To Test:
- Try enabling and disabling the analytics feature then check a team page. The feature is 50% by default
- The feature override should sync across tabs in real time and be remembered when you refresh

## Feedback I'm looking for:
By default feature state varies by user/team id. I'm only letting you force it globally. It might show enabled by default on the secret page but still be disabled on a team page.
